pkg/apis/service/v1alpha1/types.go

```diff
@@ -14,6 +14,9 @@ const CertManagementResourceNameSeed = "extension-shoot-cert-service-seed"
 // CertManagementResourceNameShoot is the name for Cert-Management resources in the shoot.
 const CertManagementResourceNameShoot = "extension-shoot-cert-service-shoot"
 
+// CertManagementResourceNameInternal is the name for internal Cert-Management resources on runtime cluster or seeds.
+const CertManagementResourceNameInternal = "extension-shoot-cert-service-internal"
```

Comment:
Does it make sense to re-use `CertManagementResourceNameSeed` for the seed case and add another one for the garden case `extension-shoot-cert-service-garden`? (see other comment about `internal`).